lib/collection/proxy-config.js

```diff
@@ -1,26 +1,32 @@
 var _ = require('../util').lodash,
     Property = require('./property').Property,
-    Url = require('./url').Url,
     UrlMatchPattern = require('../url-pattern/url-match-pattern').UrlMatchPattern,
-
-    ProxyConfig;
-
+    ProxyConfig,
+    DEFAULT_PORT = 8080,
+    PROTOCOL_HOST_SEPARATOR = '://',
+    DEFAULT_PROTOCOLS = ['http'],
```

Comment:
This is not used as a default protocol of the proxy server
That is mandatory 
This one is used when calling `getProxyUrl(testURL)`